LayoutTests/resources/SVGAnimationTestCase-testharness.js

```diff
@@ -0,0 +1,102 @@
+// NOTE(edvardt):
```

Comment:
This kind of comment isn't useful. Don't know who `edvardt` is, and a reference to "the old SVGAnimationTestCase.js" isn't useful to anyone reading this in the future.

Reply:
It is just a copy taken from Chromium so I kept it as is. If you want, I am happy to remove.

Reply:
@smfr - It is present in WPT script as well. [Link](https://github.com/web-platform-tests/wpt/blob/35283827b1658ca4856238bb798d53ece16f1445/resources/SVGAnimationTestCase-testharness.js#L4)